.github/workflows/runtime-upgrade-test.yaml

```diff
@@ -0,0 +1,157 @@
+name: Runtime upgrade test
+on:
+  issue_comment:
+    types: [created, edited]
+jobs:
+  check-permission:
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/runtime-upgrade-test')
+    runs-on: ubuntu-latest
```

Comment:
I'm a little concerned it will be slow, but let's observe it.

Reply:
According to the runs in my repo fork, it takes about an hour. Most of the time is taken by runtime release build. [Here](https://github.com/shaunxw/Astar/actions/runs/5002068823/jobs/8961568008) is an example.

Reply:
Oh, of course! We're not building the entire client binary, only the runtime. :+1: